src/lib/stitching/causality/__tests__/stitching.test.ts

```diff
@@ -5,33 +5,212 @@ import { useCausalityStitching } from "./testingUtils"
 import gql from "lib/gql"
 import schema from "schema/v1/schema"
 import v2Schema from "schema/v2/schema"
-import { runAuthenticatedQuery } from "schema/v2/test/utils"
 import { getFieldsForTypeFromSchema } from "lib/stitching/lib/getTypesFromSchema"
 
 describe("causality/stitching", () => {
-  describe("extending types", () => {
+  describe("AuctionLotStanding", () => {
     it("extends AuctionsLotStanding", async () => {
       const { types } = await useCausalityStitching()
       expect(types).toContain("AuctionsLotStanding")
     })
-  })
 
-  describe("fields", () => {
-    describe("Me", () => {
-      it("adds #auctionsLotStandingConnection to me", async () => {
-        const { getFields } = await useCausalityStitching()
-        expect(await getFields("Me")).toContain("auctionsLotStandingConnection")
+    it("resolves a SaleArtwork on an AuctionsLotStanding", async () => {
```

Comment:
Most of this is just reorganizing to match the order of whats in the `stitching` file.